tests/test_rooms/test_assign_resource.py

```diff
@@ -0,0 +1,81 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.token.token_fixture import ADMIN_TOKEN
+from fixtures.room.assign_resource_fixture import (
+    assign_resource_mutation,
+    assign_resource_mutation_response,
+    assign_resource_non_existent_room,
+    assign_non_existent_resource_id,
+    assign_excess_quantity,
+    assign_quantity_less_than_one,
+    query_string
+    )
+
+import sys
+import os
+sys.path.append(os.getcwd())
+
+
+class TestAssignResource(BaseTestCase):
+
+    def test_assign_resource(self):
+        """
+            Test for assigning a resource to a room
```

Comment:
Great work @d-kahara , I think this docstring would be more descriptive written as 'Test an admin can assign a resource to a room`